CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.4.2]
```

Comment:
Is this maybe a minor version change? Seems like re-creating is a pretty big update.

Reply:
Recreating is a larger update / issue if other things besides the bastion care about the security group. I could see this being a 0.5 update too. 🤔

Reply:
I did go ahead and bump this version - thanks.